Makefile

```diff
@@ -46,3 +54,27 @@ test-bindings: $(PKG_BINDINGS_DIR)
 
 clean-bindings:
 	rm -rf $(PKG_BINDINGS_DIR)
+
+# Generate an image for containerized builds
+# Note: This image is local only
+image: Dockerfile
+	DOCKER_BUILDKIT=1 \
```

Comment:
What is the reason for running with `BUILDKIT`? Is this available for everyone or only if you have `experimental` set up in your `daemon.josn` file?

Reply:
I had no specific reason. I was checking what `container-runtime` does. Since this is experimental, I will remove it.

Reply:
(I did hear that it was faster)